code/modules/mob/living/living.dm

```diff
@@ -296,6 +296,23 @@
 
 /mob/living/proc/revive()
 	rejuvenate()
+	if(iscarbon(src))
```

Comment:
I'd move it, and I intended to earlier, but I forgot to update it, and I figure I shouldn't be tampering with things not directly related to the thrust of the PR. This change brought this proc to the way it was before my PR came along.